src/components/AboutModal.vue

```diff
@@ -83,10 +84,11 @@ export default {
   },
   async mounted() {
     await this.$watchUntilTruly(() => this.$store.state.sdk);
-    this.nodeVersion = this.sdk.selectedNode.version;
-    this.height = await this.sdk.height();
+    this.nodeVersion = (await this.sdk.getNodesInPool()).find((node) => node.name
+      === this.sdk.selectedNodeName)?.version || '-';
```

Comment:
but this is about the node version, not the `networkId`

Reply:
my bad, proposing to use `getNodeInfo` instead of getting node info of all nodes
```suggestion
    try {
      this.nodeVersion = (await this.sdk.api.getNodeInfo()).version;
    } catch (e) {
      this.nodeVersion = '-';
    }
```